testsuite/do_tests.sh.in

```diff
@@ -249,7 +249,8 @@ run_test ()
 
     param_script="$1"
     param_success="$2"
-    param_failure="$3"
+    param_skipped="$3"
+    param_failure="$4"
```

Comment:
Please update the documentation of `run_test()` above to include `param_skipped`.